pygmt/src/meca.py

```diff
@@ -204,26 +207,52 @@ def meca(
     event_name : str or list of str, or 1-D numpy array
         Text string(s), e.g., event name(s) to appear near the beachball(s).
         List must be the same length as the number of events. Will override
-        the ``event_name`` values in ``spec`` if ``spec`` is a dictionary
+        the ``event_name`` labels in ``spec`` if ``spec`` is a dictionary
         or pd.DataFrame.
+    box : bool or str
+        [*fill*]
+        Draw a box behind the label if given via parameter or column
+        ``event_name``. Use *fill* to give a fill color [Default is
+        ``"white"``].
```

Comment:
It's also mentioned that the "label" is given by the `event_name` parameter or the `event_name` column:
```suggestion
        Draw a box behind the label if given. Use *fill* to give a fill color 
        [Default is ``"white"``].
```